Project.toml

```diff
@@ -16,15 +17,18 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 SCS = "c946c3f1-0d1f-5ce8-9dea-7daa1f7e2d13"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 TupleTools = "9d95972d-f1c8-5527-a6e0-b4b365fa01f6"
+Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
```

Comment:
Do we need Zygote as a dependency?

Reply:
We need it for the `ignore`

Reply:
ChainRulesCore has `[ignore_derivatives](https://juliadiff.org/ChainRulesCore.jl/dev/api.html#Ignoring-gradients), it should do the same thing.